src/ai/backend/manager/event_dispatcher/handlers/agent.py

```diff
@@ -112,3 +113,11 @@ async def handle_agent_error(
         event: AgentErrorEvent,
     ) -> None:
         await self._event_dispatcher_plugin_ctx.handle_event(context, source, event)
+
+    async def handle_agent_container_heartbeat(
+        self,
+        context: None,
+        source: AgentId,
+        event: AgentStatusHeartbeat,
+    ) -> None:
+        pass
```

Comment:
I understand that this is something you plan to work on in the next PR. 😄

Reply:
yes, I am going to leave a TODO comment here